packages/core/src/core/client.ts

```diff
@@ -134,6 +134,10 @@ export class GeminiClient {
     return this.chat;
   }
 
+  isInitialized(): boolean {
```

Comment:
given it's just a boolean now though it can now also be a getter property instead of a method 😄